data/types/upload/upload.go

```diff
@@ -22,9 +22,9 @@ const (
 	DeviceTagInsulinPump                 = "insulin-pump"
 	StateClosed                          = "closed"
 	StateOpen                            = "open"
-	TimeProcessingAcrossTheBoardTimezone = "across-the-board-timezone"
+	TimeProcessingAcrossTheBoardTimeZone = "across-the-board-timezone" // TODO: Rename to across-the-board-time-zone or alternative
```

Comment:
repeated const ??

Reply:
Yes, this is intentional. Slowly moving Upload to DataSet. So it is defined in both places.